### ISSUE TITLE
Opening an evidence link results in oddly sized dialog

### ISSUE THREAD

**Issue by benloh** [theRAPTLab/meme-2023#6]
1. Open a model that has a resource.
2. Click on the first evidence (next to the (1))
3. A half window is opened
![screenshot_1563](https://github.com/Vertex-Labs/meme/assets/1403057/228c96e8-8725-4eed-a6ed-bc5ec522436d)
4. It is impossible to close that window.

Four errors:
```
react.development.js:209 Warning: Failed prop type: Invalid prop `className` of type `object` supplied to `ForwardRef(Avatar)`, expected `string`.
    at Avatar (
```

```
Warning: Failed prop type: Invalid prop `className` of type `object` supplied to `ForwardRef(CardContent)`, expected `string`.
    at CardContent (
```

```
Warning: Failed prop type: Invalid prop `className` of type `object` supplied to `ForwardRef(Card)`, expected `string`.
    at Card (
```

```
Warning: Failed prop type: Invalid prop `className` of type `object` supplied to `ForwardRef(Button)`, expected `string`.
    at Button (
```

Moved from https://github.com/Vertex-Labs/meme/issues/6

**Comment by benloh** [comment]
This warnings are in `ResourceView.js`, `classes` is being imported as a javascript object.  Each css style is an object key, and the object value is itself an object.  But each components' `className` attribute expects a string.

This is going to be a problem for pretty much every MUI component.

Theoretically we can use the `styled` utility to replace `makeStyles` and `withStyles`, but this doesn't help us disentangle ourselves from MUI.  In fact, we would end up spending time converting to MUI5 first.

According to ChatGPT:
```

The error Invalid prop 'className' of type 'object' supplied to 'ForwardRef(Chip)', expected 'string' typically arises because of a change in how class names are handled between MUI v4 and v5. The core cause is that in MUI v5, the styling system underwent significant changes, including how class names are passed and managed.

In MUI v4, it was common to use the makeStyles hook or withStyles HOC to create classes, which were then passed as objects. However, in MUI v5, the recommended approach is to use the new sx prop and the styled utility, and the class names need to be strings rather than objects.
```

We might be better off stripping all the MUI components and rebuilding it in pieces.

**Comment by dsriseah** [comment]
My gut feeling is to look one level up and look at the source style definitions in meme-styles that are propagating through the old styled-component transformation chain that produced the expected classname. If you don't know the mechanism of what it is doing, you might be missing out on an easier conversion path than if you're just concentrating on React errors on the component level.

**Comment by benloh** [comment]
Fixed with #25
